opencti-platform/opencti-graphql/src/resolvers/stixRefRelationship.js

```diff
@@ -25,6 +26,7 @@ const stixRefRelationshipResolvers = {
     stixRefRelationships: (_, args, context) => findAll(context, context.user, args),
     stixNestedRefRelationships: (_, args, context) => findNested(context, context.user, args),
     stixSchemaRefRelationships: (_, { id, toType }, context) => schemaRefRelationships(context, context.user, id, toType),
+    stixSchemaRefRelationshipsPossibleTypes: (_, { type }, __) => schemaRefRelationshipsPossibleTypes(type),
```

Comment:
Could you please add a test on this new resolver API ?

Reply:
done :)